public/index.html

```diff
@@ -0,0 +1,76 @@
+     <ul>
+         <li>Name:Lungu</li>
+         <li>Surname:Constantin</li>
+         <li>Age:17</li>
+         <li>Country:Moldova</li>
+         
+     </ul>
+ </header>
+ <main>
+     
+         <section id="about"><h2>About</h2>
+            <img src="https://media.istockphoto.com/photos/child-hands-formig-heart-shape-picture-id951945718?k=6&m=951945718&s=612x612&w=0&h=ih-N7RytxrTfhDyvyTQCA5q5xKoJToKSYgdsJ_mHrv0=" alt="">
+<p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nam voluptates animi placeat nesciunt porro, ex sapiente nobis minus voluptatibus obcaecati odit? Quibusdam temporibus provident velit enim molestias ab odio, recusandae ea, repellat ullam alias delectus voluptatibus odit quae! Aliquid dolorem adipisci pariatur est nobis suscipit quo placeat autem earum eligendi! Lorem ipsum dolor sit amet consectetur adipisicing elit. At nam, architecto aut quidem cupiditate totam? Vero, libero nam! Deleniti inventore ducimus nobis sit saepe velit dolores reiciendis quam aliquam, veniam omnis at maiores expedita magni natus eaque rerum sunt laudantium recusandae enim. Quo, soluta? Culpa optio provident consequatur, est eius aspernatur harum, possimus ipsum excepturi laborum nesciunt et, earum ex architecto expedita numquam? Officia non aspernatur laboriosam consequatur maiores quisquam illum adipisci natus velit optio, cumque fugit dolorum laudantium incidunt voluptatem omnis expedita porro exercitationem mollitia quibusdam consequuntur, alias temporibus quam. Qui odio itaque, harum ullam voluptatum corrupti iusto magni?</p>
+        </section>
+        <section>
+            <h2>
+                Career Objective
+            </h2>
+            <p>MA Student in English at Stanford University, made the Dean’s List for three consecutive years (2015-2017), with two study abroad experiences and a semester-long research internship in Oxford,<br> UK looking to use my strong research and writing skills,br as well as my expertise in contemporary literature in the position of Editorial Assistant at Penguin Random House.</p>
+        </section>
+        
+         <section id="skills"> <h2>Skills</h2>
+            <!-- I KNOW THIS MUST BE USED WITH LI,BUT IT 
+            LOOKS COOLER ONLY IF YOU USE ONLY OL -->
+            <ol>
+            <li>Fast typing</li>
+            <li>HTML and CSS knowledge</li>
+            <li>English Language </li>
+            <li>Imaginative</li>
+            <li>Careful</li>
+            <li>Make Rubiks Cube</li>
+            <li>Good Memory</li>
+        </ol>
+        </section>
+         <section id="exp"><h2>Experience</h2>
+        <ol>
+            <li>Assisted in hiring and managing a pool of freelancers as needed, including development editors,
+                <br> copy editors, proofreaders, indexers, recipe testers, and technical editors, including remote and on-site employees.</li><br>
+            <li>Assisted in the development, design, and preparation of sales materials.</li><br>
```

Comment:
Element ```br``` is not allowed here 
If you want to provide one empty line, use it inside of ```li``` tag